furms-core/src/main/java/io/imunity/furms/core/config/security/rest/PresetUser.java

```diff
@@ -28,8 +26,8 @@
 			password,
 			authorities,
 			FURMSUser.builder()
-				.id(new PersistentId(username)) //FIXME FU-104 KB: I'm afraid this can cause problems. Let's discuss if unclear why.
 				.firstName("Central Idp")
+				.email("centralIdp@email.com")
```

Comment:
If I remember correctly requirements said that email is required, so I put here some placeholder, but I don't know what mail should be really here.